Elevato/Lift.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Elevato.Models;
+using Elevator.Extand;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevator
+{
+    public class Lift : ILift
+    {
+        private int _CurrentFloor;
+        private readonly List<int> _path;
+
+        public volatile bool IsMoving;
+        public volatile Direction direction;
+
+        public EventHandler<int> IsMoved;
+
+        public Lift()
+        {
+            _CurrentFloor = 3;
+            _path = new List<int>();
+        }
+
+        public void Request(Message message)
+        {
+            if (_path.Contains(message.FloorNumber))
+            {
+                if (message.Direction == direction.ToString())
+                    _path.Insert(_path.IndexOf(message.FloorNumber), message.FloorNumber);
+            }
+            else
+                AddStop(message);
+        }
+
```

Comment:
Private method - this should be in another class and made public

Reply:
not sure which privet methods are you taking about